local-running/setConfigForLocalOrCloudRunning.py

```diff
@@ -194,7 +194,8 @@ def get_local_running_params(environment, client_id_suffix):
             "tokenUrl":"http://host.docker.internal:3015/token",
             "credentialUrl":"http://host.docker.internal:3015/credentials/issue",
             "clientId":"ipv-core-dev{client_id_suffix}",
-            "signingKey":"{{\\"kty\\":\\"EC\\",\\"crv\\":\\"P-256\\",\\"x\\":\\"RBXnILIdExUEWUJMlYeD6agE8u9gGgA3InKrd5TKhhY\\",\\"y\\":\\"kKtt9v_xq9oqvv5_E8AHcV77IYQfyNwaTQyTYxdO_UM\\"}}",
+            "signingAlgorithm": "RSA",
```

Comment:
It would've been ok since there would've been a default to EC. I'm now no longer storing the algorithm in the cir config anyway, since it can be determined from the signing key header